chocs/http_response.py

```diff
@@ -39,6 +40,9 @@ def writable(self):
     def close(self):
         self.body.close()
 
+    def set_cookie(self, cookie: Cookie):
+        self.headers.add_header(*cookie.header())
```

Comment:
This method's signature has changed and now valid one should look like the following:
`self.headers.set(*cookie.header())`